python/openchange/provision.py

```diff
@@ -739,6 +724,22 @@ def newuser(names, lp, creds, username=None, mail=None):
         print "[!] User '%s' not found" % username
 
 
+def delete_user(names, lp, creds, username=None):
+    db = get_local_samdb(names, lp, creds)
+    basedn = "CN=Users,%s" % names.domaindn
+    user_dn = get_user_dn(db, basedn, username)
+    if not user_dn:
+        print "[!] User '%s' not found" % username
+        return
+
+    to_delete = ['mailNickName', 'homeMDB', 'homeMTA',' legacyExchangeDN',
+                 'proxyAddresses', 'msExchUserAccountControl',
+                 'msExchRecipientTypeDetails', 'msExchRecipientDisplayType']
+    _delete_attrs_ignore_no_existent(db, user_dn, to_delete)
```

Comment:
Hi, I'm grammar nazi, how is your day? this should be `ignore_nonexistent`, `no_existent` is wrong